docs/translations/translations.json

```
@@ -33,7 +33,7 @@
   "tableOfContents": "Contents",
   "component": "Component Kit",
   "mdbProDescr": "Material Dashboard Pro React is a premium Material-UI admin theme.",
-  "mkProDescr": "A Badass Material-UI Kit based on Material Design.",
+  "mkProDescr": "A Material-UI Kit based on Material Design.",
```

```suggestion
  "mkProDescr": "An awesome Material-UI Kit based on Material Design.",
```

(No issue having a superlative here, just something other than "badass" :) )